it/controllers/CalculationControllerISpec.scala

```diff
@@ -387,12 +395,210 @@ class CalculationControllerISpec extends IntegrationSpecCommonBase {
     )
   )
 
+  val samplePenaltyDetails: PenaltyDetails = PenaltyDetails(
+    totalisations = Some(Totalisations(
+      LSPTotalValue = 200,
+      penalisedPrincipalTotal = 2000,
+      LPPPostedTotal = 165.25,
+      LPPEstimatedTotal = 15.26,
+      LPIPostedTotal = 1968.2,
+      LPIEstimatedTotal = 7)),
+    lateSubmissionPenalty = Some(
+      LateSubmissionPenalty(
+        summary = LSPSummary(
+          activePenaltyPoints = 10,
+          inactivePenaltyPoints = 12,
+          regimeThreshold = 10,
+          penaltyChargeAmount = 684.25
+        ),
+        details = Seq(LSPDetails(
+          penaltyNumber = "12345678901234",
+          penaltyOrder = "01",
+          penaltyCategory = LSPPenaltyCategoryEnum.Point,
+          penaltyStatus = LSPPenaltyStatusEnum.Active,
+          FAPIndicator = "X",
+          penaltyCreationDate = LocalDate.parse("2069-10-30"),
+          penaltyExpiryDate = LocalDate.parse("2069-10-30"),
+          expiryReason = Some("FAP"),
+          communicationsDate = LocalDate.parse("2069-10-30"),
+          lateSubmissions = Some(Seq(
+            LateSubmission(
+              taxPeriodStartDate = Some(LocalDate.parse("2069-10-30")),
+              taxPeriodEndDate = Some(LocalDate.parse("2069-10-30")),
+              taxPeriodDueDate = Some(LocalDate.parse("2069-10-30")),
+              returnReceiptDate = Some(LocalDate.parse("2069-10-30")),
+              taxReturnStatus = TaxReturnStatusEnum.Fulfilled
+            )
+          )),
+          appealInformation = Some(Seq(
+            AppealInformationType(
+              appealStatus = Some(AppealStatusEnum.Unappealable),
+              appealLevel = Some("01")
+            )
+          )),
+          chargeAmount = Some(200),
+          chargeOutstandingAmount = Some(200),
+          chargeDueDate = Some(LocalDate.parse("2069-10-30"))
+        ))
+      )
+    ),
+    latePaymentPenalty = Some(v3LatePaymentPenalty(
+      details = Seq(LPPDetails(
+        principalChargeReference = "12345678901234",
+        penaltyCategory = LPPPenaltyCategoryEnum.LPP1,
+        penaltyStatus = LPPPenaltyStatusEnum.Accruing,
+        penaltyAmountPaid = Some(277.00),
+        penaltyAmountOutstanding = Some(123.00),
+        LPP1LRDays = Some("15"),
+        LPP1HRDays = Some("31"),
+        LPP2Days = Some("31"),
+        LPP1LRCalculationAmount = Some(123.00),
+        LPP1HRCalculationAmount = Some(123.00),
+        LPP2Percentage = Some(2.00),
```

Comment:
Could you change this to:
```suggestion
        LPP2Percentage = Some(4.00),
```